agent/xds/rbac.go

```diff
@@ -580,37 +637,52 @@ func idPrincipal(src structs.ServiceName) *envoy_rbac_v3.Principal {
 	}
 }
 
-func makeSpiffePattern(sourceAP, sourceNS, sourceName string) string {
-	if sourceNS == structs.WildcardSpecifier && sourceName != structs.WildcardSpecifier {
-		panic(fmt.Sprintf("not possible to have a wildcarded namespace %q but an exact service %q", sourceNS, sourceName))
+const anyPath = `[^/]+`
+
+func makeSpiffePattern(src rbacService) string {
+	var (
+		host = anyPath
```

Comment:
```suggestion
		host = anyPath // TrustDomain is not verified by RBAC, so we match on any value.
```
or something up here for now

Reply:
Added a todo to use local trust domain